lms/services/roster.py

```diff
@@ -80,29 +80,25 @@ def get_assignment_roster(
         role_scope: RoleScope | None = None,
         role_type: RoleType | None = None,
         h_userids: list[str] | None = None,
-    ) -> Select[tuple[LMSUser]]:
+    ) -> Select[tuple[LMSUser, bool]]:
         """Get the roster information for a course from our DB."""
         roster_query = (
-            select(AssignmentRoster.lms_user_id)
-            .join(LTIRole)
-            .where(
-                AssignmentRoster.assignment_id == assignment.id,
-                AssignmentRoster.active.is_(True),
-            )
-        )
+            select(LMSUser, AssignmentRoster.active)
+            .join(LMSUser, AssignmentRoster.lms_user_id == LMSUser.id)
+            .join(LTIRole, AssignmentRoster.lti_role_id == LTIRole.id)
+            .where(AssignmentRoster.assignment_id == assignment.id)
+        ).distinct()
```

Comment:
As we now use a join we have to return distinct rows.

In the spec is not possible to get a user active with one role but inactive with another so we'll always get unique users here.